templates/LeftAndMain_Menu.ss

```diff
@@ -22,12 +22,12 @@
 			<% include SubsiteList %>
 		<% end_if %>
 	</div>
-		
+
 	<div class="cms-panel-content center">
 		<ul class="cms-menu-list">
 		<% loop $MainMenu %>
 			<li class="$LinkingMode $FirstLast <% if $LinkingMode == 'link' %><% else %>opened<% end_if %>" id="Menu-$Code" title="$Title.ATT">
-				<a href="$Link" <% if $Code == 'Help' %>target="_blank"<% end_if %>>
+				<a href="$Link" <% if $Code == 'Help' %>target="_blank"<% end_if %> $AttributesHTML>
```

Comment:
In the core template it leaves out the target="_blank",since this was moved to the $AttributesHTML.

How about we change this to `<a href="$Link" $AttributesHTML>` as per the main template?